tests/by-util/test_cp.rs

```diff
@@ -1388,6 +1388,52 @@ fn test_closes_file_descriptors() {
         .succeeds();
 }
 
+#[cfg(any(target_os = "linux", target_os = "android"))]
+#[test]
+fn test_cp_reflink_always_override() {
+    let scene = TestScenario::new(util_name!());
+
+    // Test must be run as root (or with `sudo -E`)
```

Comment:
Can you do like in
https://github.com/uutils/coreutils/blob/40095e1b5070b1c353de613f0b258a030b978407/tests/by-util/test_chown.rs#L432-L442 instead?

thanks

Reply:
I am not sure this allows me to do what I want, only the tested command (`cp`) can be run as root with `run_ucmd_as_root`, while we need only need root for `mount`/`umount` in this test. Nevertheless, I tried to take into account this idea to allow running the test as non root (running `sudo` only for `mount` and `umount`)